src/components/vue/GrammySwitch.vue

```diff
@@ -1,7 +1,7 @@
 <script setup lang="ts">
 import { withDefaults } from 'vue';
 
-withDefaults(defineProps<{ 'modelValue': boolean, id: string }>(), { 'modelValue': false })
+withDefaults(defineProps<{ modelValue: boolean, id: string }>(), { 'modelValue': false })
```

Comment:
Single quotes?

Reply:
I wonder why prettier didn't correct this 🤔